src/sql/workbench/contrib/connection/browser/connectionStatus.ts

```diff
@@ -59,16 +64,38 @@ export class ConnectionStatusbarItem extends Disposable implements IWorkbenchCon
 	// Update the connection status shown in the bar
 	private _updateStatus(): void {
 		let activeConnection = TaskUtilities.getCurrentGlobalConnection(this._objectExplorerService, this._connectionManagementService, this._editorService, this._logService);
+		let id = undefined;
+		if (this._editorService.activeEditor) {
```

Comment:
nit: Please add comment explaining what this section of code is doing - it's not immediately obvious for someone not familiar with the feature being added.